lib/config/presets/github/index.ts

```diff
@@ -15,9 +15,15 @@ const http = new GithubHttp();
 export async function fetchJSONFile(
   repo: string,
   fileName: string,
-  endpoint: string
+  endpoint: string,
+  packageTag: string
 ): Promise<Preset> {
-  const url = `${endpoint}repos/${repo}/contents/${fileName}`;
+  let ref = '';
+  if (packageTag != null) {
```

Comment:
Is there a possibility of `undefined` slipping in here?

Reply:
I didn't knew about this, I'm a newcomer to TS :sweat_smile: I updated accordingly